Makefile

```diff
@@ -256,11 +258,17 @@ status:
 checkout:
 	@for r in $(REPOS) ; do \
 		cd $$r; \
-		git show-ref --verify --quiet refs/heads/$(branch); \
+		git fetch 2>/dev/null; \
+		if [ $$? -ne 0 ]; then \
+			echo "$(YELLOW)Fetching the latest from the remote failed, you may not be able to checkout an existing remote branch.$(NO_COLOR)"; \
```

Comment:
I think we should not introduce visual noise as a side-effect to a PR. 
If we want to use color in our process we should probably introduce this more intentionally (particularly since two people on engineering/product staff have color blindness)

Reply:
ah, I did not know this. Removed. Thanks.